cluster/src/main/java/org/apache/iotdb/cluster/server/handlers/caller/AppendNodeEntryHandler.java

```diff
@@ -66,7 +67,7 @@ public void onComplete(Long response) {
         logger.debug("{}: Received an agreement from {} for {}, remaining votes to succeed: {}",
             member.getName(), receiver, log, remaining);
         if (remaining == 0) {
-          logger.debug("{}: Log {} is accepted by the quorum", member.getName(), log);
+          logger.debug("{}: Log {} is accepted by the quorum {}", member.getName(), log, log.getCurrLogIndex());
```

Comment:
Adding the parameter after "quorum" is a bit confusing, maybe we can use:
`"{}: Log[{}] {} is accepted by the quorum", member.getName(), log.getCurrLogIndex(), log`